README.md

```diff
@@ -171,6 +171,8 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
 
   * https://www.simplybuilt.com - SimplyBuilt offers free website building and hosting for open source projects (http://www.simplybuilt.com/explore/free-websites-for-open-source-projects). Simple alternative to GitHub Pages.
   * http://www.devport.co - Turn GitHub projects, Apps, and websites into a personal developer portfolio.
+  * https://pages.github.com - Websites for you and your projects, hosted directly from your GitHub repository. Just edit, push, and your changes are live.
```

Comment:
would rather not list the same thing many times, if you like maybe expand the existing github explanation to mention it also does this?